src/views/Events.vue

```diff
@@ -8,7 +8,8 @@
           v-for="event in futureEvents"
           :key="event.id"
           :title="event.title"
-          :subtitle="event.desc"
+          :subtitle="event.subtitle"
+          :desc="event.desc"
```

Comment:
ListCard doesn't actually use the description right? My understanding is that the description is only used by the EventModal, and the subtitle is used by the ListCard. If that's the case, we don't need to give the "desc" prop to ListCard (and can remove it from ListCard's prop validation).